docs/recipes/docker-server/README.md

```diff
@@ -65,4 +65,21 @@ kubectl get service
 # lhci-server   LoadBalancer   10.X.X.X       X.X.X.X        80:XXXXX/TCP   2m
 ```
 
+### Docker Compose
+
+You can create a docker-compose.yml file. Execute `docker-compose up` relative to the file's location.
```

Comment:
an explanation of why you'd want to use docker-compose for this would be great.

AFAICT, there's no real benefit to using it on its own because the lhci server is just a single docker container and there's not really much to orchestrate. `docker container run --publish 9001:9001 --mount='source=lhci-data,target=/data' --detach lhci` accomplishes the same task, no?

Reply:
Admittedly, this approach doesn't have a huge benefit but it will save some people time:

1. Docker-compose should already be available to anyone using docker locally.
2. This will save users on Windows machines from having to slightly modify the repository to get it working (this is a two-step ordeal I can detail further if you'd like).
3. This snippet could be used verbatim in an Azure DevOps App Service or similar.

Reply:
Ah 2 and 3 are great reasons (would love to hear more about 2 separately as well!) mind adding 3 to the comment?

Maybe just...
```suggestion
You can create a docker-compose.yml file to orchestrate an LHCI server in cloud services or along with your existing app docker containers. Execute `docker-compose up` relative to the file's location.
```

Reply:
Yes, very nice suggestion!